test/unit/registration.js

```diff
@@ -589,3 +590,48 @@ exports['trigger param configuration parse failure for invalid JSON propagates t
         test.done();
     });
 };
+
+exports['addMessage prepops and passes the right information to messages.addMessage'] = (test) => {
+    const testPhone = '1234',
+          testMessage = 'A Test Message',
+          testRegistration = 'some registrations',
+          testPerson = 'a patient contact';
```

Comment:
Change this to an object with a name property so you can test the `patient_name` alias works as expected.